csi/build.rs

```diff
@@ -4,5 +4,9 @@ fn main() {
     tonic_build::configure()
         .build_server(true)
         .compile(&["proto/csi.proto"], &["proto"])
-        .unwrap_or_else(|e| panic!("csi protobuf compilation failed: {}", e));
+        .expect("csi protobuf compilation failed.");
```

Comment:
minor nit: should leave out the full stop at the end, as the message in the panic will follow this with a ':'